basics_of_sst-core/tutorial/Example01.md

```diff
@@ -0,0 +1,436 @@
+# Communicating Between Two Components
+
+Communication between components in SST is accomplished by passing events over links that connect the components.  This example will demonstrate how to send messages between two components in SST.  To show that the components are really communicating the component clock event handler will be modified to increment the clock tick count when a message is received and end the simulation when the specified number of clock counts have been received.
+
+For this example we'll need to make the following changes.
+
+1.  Add a link to the component over which the message will be sent.
+2.  Define the message to be sent.
+3.  Update the clock event handler to process the incoming messages.
+4.  Configure the simulation to connect two components via the link.
```

Comment:
Change to "Configure the simulation with two components and connect the two components' ports via a link."